ccd-definition/CaseEventToComplexTypes/respondents/enterRespondents.json

```diff
@@ -412,5 +412,13 @@
     "CaseFieldID": "respondents1",
     "ListElementCode": "solicitor.regionalOfficeAddress.Country",
     "DisplayContext": "OPTIONAL"
+  },
+  {
+    "ID": "EnterRespondents",
+    "CaseEventID": "enterRespondents",
+    "CaseFieldID": "respondents1",
+    "ListElementCode": "activeParty",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "activeParty=\"*\" OR activeParty!=\"*\""
```

Comment:
we use stuff like
```
    "FieldShowCondition": "party!=\"SHOW_LABEL\""
```
further up the file, want to use something like that instead as it is probs less for whatever is computing the show hide evaluate and therefore have a slight (probably minor) impact on performance